src/shared/install.c

```diff
@@ -370,6 +370,7 @@ void install_changes_dump(int r, const char *verb, const InstallChange *changes,
                                          changes[i].path);
                         break;
                 case INSTALL_CHANGE_IS_DANGLING:
+                case -ENOLINK:
```

Comment:
See the comment above:
`                /* When making changes here, make sure to also change install_error() in dbus-manager.c. */`

:]